### ISSUE TITLE
Implement Naive Bayes model in Python

### ISSUE THREAD

**Issue by arya2004** [arya2004/credit-score-classification#7]
### 🎯 Task
Convert the existing Naive Bayes model into **Python** using `sklearn.naive_bayes`.

✅ Expected:
- Similar logic to `naive_bayes.ipynb`
- Evaluate accuracy, precision, recall

📁 Save as: `src/naive_bayes_python.ipynb`

**Comment by Shwetank-Maurya** [comment]
Hey You can assign this task to me I want to work on this repo end to end>>
